tests/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Service Worker Toolbox Tests</title>
+    <link rel="stylesheet" href="//code.jquery.com/qunit/qunit-1.17.0.css"/>
```

Comment:
Can you pull in `qunit` locally via a `devDevependency` in your `package.json` and use that? I'm sure `code.jquery.com` isn't going away anytime soon, but I don't see the benefit of depending on something remote.